interface/globals.php

```diff
@@ -98,7 +98,7 @@ function undoMagicQuotes($array, $topLevel=true) {
     $tmp = $_GET['site'];
   }
   else {
-    if (!$ignoreAuth) die("Site ID is missing from session data!");
+    if (!isset($ignoreAuth)) die("Site ID is missing from session data!");
```

Comment:
I think using empty would be more accurate here(and should not throw a php error, although should test it):
``````
if (empty($ignoreAuth)) die("Site ID is missing from session data!");